hooks/useComputedValues.ts

```
@@ -0,0 +1,172 @@
+  if (typeof config.max === "object" && config.max._type === "computed") {
+    dependencies.push(...extractDependencies(config.max.vars));
+  }
+
+  return Array.from(new Set(dependencies));
+}
+
+function computeMinMax(
+  config: ComputedConfig,
+  data: Record<string, any>
+): { min: number; max: number } {
+  let min: number = -Infinity;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)